infinite_tracing/lib/infinite_tracing/connection.rb

```diff
@@ -114,7 +114,7 @@ def notify_agent_started
           @agent_connected = true
           @agent_started.signal
         end
-        @active_clients.each_value(&:restart)
+        @active_clients.values.each(&:restart)
```

Comment:
This change is actually fixing a real error that can actually occur with infinite tracing that i found while running the tests over and over. The old way this was written it was possible for a `can't add a new key into hash during iteration` error to get raise when this line was happening and record_spans was also called.